pages/common/imapsync.md

```diff
@@ -0,0 +1,7 @@
+# imapsync
+
+> Email IMAP tool for syncing, copying and migrating email mailboxes between two imap servers, one way, and without duplicates.
+
+- Synchronize imap account between host1 and host2:
```

Comment:
We usually start with simpler command invocations and then move on to complex ones. Is there a simple `imapsync` that users can invoke ? Or something with lesser no. of flags, and then mention this example.

Reply:
Nope, unfortunately this is the simplest. Considering what the command does, how else would it know which server and account to sync? :)

Reply:
Yes, I thought that too. Just wanted to check :)